content/en/variables/git.md

```diff
@@ -49,3 +49,24 @@ The `GitInfo` object contains the following fields:
 If the `.GitInfo` feature is enabled, `.Lastmod` (on `Page`) is fetched from Git i.e. `.GitInfo.AuthorDate`. This behavior can be changed by adding your own [front matter configuration for dates](/getting-started/configuration/#configure-front-matter).
 
 [configuration]: /getting-started/configuration/
+
+## Hosting Considerations
+
+On the site host, your repository must be "deep-cloned," so the returned `.GitInfo` data will be accurate. Otherwise, your site may display only data from your latest commit. Where it's not possible to configure a host's cloning depth, you must handle this through CI/CD (*e.g.*, a 
+GitHub Action or GitLab CI/CD). See the following table:
+
+| Hosting service | Clone depth | Configurable? |
+| :-------------- | :---------: | :-----------: |
+| Cloudflare Pages | Shallow | ✔️ [^CFP] |
+| DigitalOcean App Platform | Deep | ❌ |
+| GitHub Pages | Shallow | ✔️ [^GHP] |
+| GitLab Pages | Shallow | ✔️ [^GLP] |
+| Netlify | Deep | ❌ |
+| Render | Shallow | ❌ |
+| Vercel | Shallow | ❌ |
+
+[^CFP]: To configure a Cloudflare Pages site for deep cloning, preface the site's normal Hugo build command with `git fetch --unshallow &&` (*e.g.*, `git fetch --unshallow && hugo`).
+
+[^GHP]: You can configure the GitHub Runner to do a deep clone by specifying `fetch-depth: 0` in the applicable "checkout" step of your workflow file, as shown in the Hugo documentation's [example workflow file](/hosting-and-deployment/hosting-on-github/#procedure).
```

Comment:
```suggestion
[^GHP]: You can configure the GitHub Action to do a deep clone by specifying `fetch-depth: 0` in the applicable "checkout" step of your workflow file, as shown in the Hugo documentation's [example workflow file](/hosting-and-deployment/hosting-on-github/#procedure).
```

Reply:
Made the change directly in the file rather than accepting the suggestion because I still needed to add the GL-related Hugo link you mentioned earlier. Done.